welcome/supported-environments.adoc

```diff
@@ -0,0 +1,24 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="supported-environments"]
+= Supported environments
+:context: supported-environments
+
+toc::[]
+
+Currently {product-title} is supported on the following architectures:
+
+* ARM
```

Comment:
Maybe you can reach out to Duncan on the best way to referece these.

S390Z, PowerPC, AMD64, or 64-bit x86, ARM all seem to be different.